ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace HttpClientExample
+{
+	public class List
+	{
+		public string name { get; set; }
+	}
+	public class Email
+	{
+		public string email { get; set; }
+	}
+	public class Campaign
+	{
+		public string name { get; set; }

```

Comment:
@cristiangiagante  said:
>  Hi guys, let me know if I'm wrong but it 's supposed that public properties must be writted with PascalCase:
https://msdn.microsoft.com/en-us/library/ms229043(v=vs.100).aspx

You are right, but in this case, since these classes are representing JSON objects, I think that it is fine having _camelCase_, another alternative is configuring JSON.NET to change the case on serializing, or using annotations. But, I personally prefer this approach, it is simple and I do not see any drawback.